Examples/Python/python/acts/examples/itk.py

```diff
@@ -434,7 +434,7 @@ def itkSeedingAlgConfig(inputSpacePointsType: InputSpacePointsType):
         seedConfirmationFilter = True
         impactWeightFactor = 100
         compatSeedLimit = 3
-        numSeedIncrement = float("inf")
+        numSeedIncrement = 100
```

Comment:
is this intended? why are we changing it?

Reply:
we never reach 100 compatible seeds for PPP at mu200 so I think this doesn't necessarily need to be set to infinity, but I can revert to `float("inf")` if you prefer

Reply:
better lower numbers for the memory, right?